assets/app/features/upgrade/controllers/upgrade7-upgrade-admin.controller.js

```diff
@@ -0,0 +1,66 @@
+(function() {
+    'use strict';
+
+    /**
+    * @ngdoc function
+    * @name crowbarApp.controller:Upgrade7UpgradeAdminController
+    * @description
+    * # Upgrade7UpgradeAdminController
+    * This is the controller used on the Upgrade Admin Server page
+    */
+    angular.module('crowbarApp')
+        .controller('Upgrade7UpgradeAdminController', Upgrade7UpgradeAdminController);
+
+    Upgrade7UpgradeAdminController.$inject = ['$timeout', 'upgradeUpgradeAdminFactory'];
+    // @ngInject
+    function Upgrade7UpgradeAdminController($timeout, upgradeUpgradeAdminFactory) {
+        var vm = this;
+        vm.adminUpgrade = {
+            completed: false,
+            running: false,
+            beginAdminUpgrade: beginAdminUpgrade,
+            checkAdminUpgrade: checkAdminUpgrade
```

Comment:
I'm not sure we need to expose `checkAdminUpgrade` function through the vm. What do you think?